shared/storage.ts

```diff
@@ -28,6 +29,14 @@ class Storage {
     return await MMKV.getItem(key);
   };
 
+  static isVCStorageInitialised = async (): Promise<boolean> => {
```

Comment:
can we use async / await instead of Promise.then() to be consistant with the other methods in the storage.ts